### DISCUSSION TITLE
Custom badge

### DISCUSSION THREAD

**Discussion by alexnix** [vitorhugods/AvatarView#26] [Ideas]
Is there a way to load an actual image for the badge?

**Comment by vitorhugods** [comment]
Only solid colours are currently supported.

I guess it would be possible to accept drawables and modifying the `onDraw` function to draw it instead of solid colours.

At the moment I have no plans in adding new features to this library, but I am thinking about re-implementing it using Compose and making it more customisable.

Haven't taken any actions towards this yet.

**Comment by alexnix** [comment]
I guess there is also the option to wrap `AvatarView` and `ImageView` in a `RelativeLayout` and get this done manually. Haven't thought this option when I posted the question, nevertheless it would probably be cool to be part of the library, I think
